kahuna/public/js/components/gr-image-metadata/gr-image-metadata.js

```diff
@@ -40,6 +40,28 @@ module.controller('grImageMetadataCtrl', [
     ctrl.metadata = ctrl.image.data.metadata;
     ctrl.identifiers = ctrl.image.data.identifiers;
 
+    mediaApi.fileMetadata(ctrl.image).then(resource => {
+      return resource.data;
+    }).then(fileMetadata => {
+      const groupedConfigsByDirectory = window._clientConfig.fileMetadataConfig.reduce((r, a) => {
```

Comment:
Rather than read this here every time it is used it should be parsed once in a controller. e.g. https://github.com/guardian/grid/blob/3b03881f9aa9a6331bb34c99967712e842540998/kahuna/public/js/errors/global.js#L44

Reply:
Removed this bit that's now irrelevant. The visible fileMetadata fields are accessible within `ctrl.image` variable hence no need to fetch `fileMetadata` via the `mediaApi`